Lib/test/test_configparser.py

```diff
@@ -720,6 +721,11 @@ def test_read_returns_file_list(self):
         parsed_files = cf.read(file1)
         self.assertEqual(parsed_files, [file1])
         self.assertEqual(cf.get("Foo Bar", "foo"), "newbar")
+        # check when we pass only a Path object:
+        cf = self.newconfig()
+        parsed_files = cf.read(pathlib.Path(file1))
+        self.assertEqual(parsed_files, [pathlib.Path(file1)])
```

Comment:
Good point. Anyway I'm not too worried about `bytes` paths at the moment. We can fix all corner cases if we decide to support `bytes` in 3.7.